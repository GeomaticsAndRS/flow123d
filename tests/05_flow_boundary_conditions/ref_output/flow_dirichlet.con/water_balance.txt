	"time"	"region"	"quantity"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"plane"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	".bc_south"	"water_volume"	0.499985	0	0.499985	0	0	0	0	0	0	0
	0	".bc_east"	"water_volume"	-0.499992	-0.499992	0	0	0	0	0	0	0	0
	0	".bc_north"	"water_volume"	-0.500004	-0.500004	0	0	0	0	0	0	0	0
	0	".bc_west"	"water_volume"	0.500011	0	0.500011	0	0	0	0	0	0	0
