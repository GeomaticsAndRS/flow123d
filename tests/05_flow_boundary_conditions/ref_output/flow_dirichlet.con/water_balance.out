# ----------------------------------------------- MASS BALANCE -----------------------------------------------
# Time: 0


# Mass flux through boundary [M/T]:
# [boundary_id] [label]                  [substance]   [total flux]  [outward flux][inward flux] 
# -------------------------------------------------------------------------------------------------------------
  2             .bc_south                water_volume  0.499985      0.499985      0             
  3             .bc_east                 water_volume  -0.499992     0             -0.499992     
  4             .bc_north                water_volume  -0.500004     0             -0.500004     
  5             .bc_west                 water_volume  0.500011      0.500011      0             
# -------------------------------------------------------------------------------------------------------------
# Total mass flux of substance [M/T]     water_volume  -5.27743e-08  0.999996      -0.999996     


# Mass [M] and sources [M/T] on regions:
# [region_id]   [label]                  [substance]   [total_mass]  [total_source]
# -------------------------------------------------------------------------------------------------------------
  1             plane                    water_volume  0             0             
# -------------------------------------------------------------------------------------------------------------
# Total mass [M] and sources [M/T]       water_volume  0             0             


